cerberus-web/build.gradle

```diff
@@ -15,7 +15,7 @@
  */
 
 plugins {
-  id 'org.springframework.boot' version "${springBootVersion}"
+  id 'org.springframework.boot' version "2.4.0"
```

Comment:
I think we should keep this to `"${springBootVersion}"` so that way it stays in sync with the versioning in `build.gradle` at the root.

Reply:
I don't disagree, Dependabot didn't do the spring upgrade when it was set that way which is why I made the change. If we are fine excluding Spring from automatic upgrades we can switch it back but it's probably worth having a discussion.

Reply:
Okay sounds good, let's discuss it tomorrow